lib/sidekiq/cron/poller.rb

```diff
@@ -7,12 +7,12 @@ module Sidekiq
   module Cron
     # The Poller checks Redis every N seconds for sheduled cron jobs.
     class Poller < Sidekiq::Scheduled::Poller
-      def initialize(options = {})
+      def initialize(config = nil)
```

Comment:
Alternative is to try have default be dynamic between {} & `Sidekiq::Config.new` or `Sidekiq.default_configuration`. But I think it's better to match the underlying method, where this parameter is non-optional. So it's a mistake to call this without a parameter 6.5 & beyond. So maybe an argument error should be raised when config is nil in >=6.5 case